### ISSUE TITLE
dar formato a la información actual

### ISSUE THREAD

**Issue by MiaFate** [Akatsuki-no-Dev/akatsuki-no-clima#4]
poner la info que muestra ahora en una card por ej

![image](https://user-images.githubusercontent.com/52300099/125800134-534d2892-d16e-41ef-9a78-4b3f4a418258.png)

**Comment by andyCasa** [comment]
card tipo un div contenedor?

**Comment by andyCasa** [comment]
medio estilizado con css?

**Comment by MiaFate** [comment]
claro, asi como hiciste por las dudas dejo aca por si alguien quiere chusmear 
https://www.w3schools.com/howto/howto_css_cards.asp
